client/src/sass/player/peertube-skin.scss

```diff
@@ -88,6 +88,10 @@ body {
     &:hover {
       opacity: 0.8;
     }
+
+    &.focus-visible{
+      background-color: var(--mainColor);
+    }
```

Comment:
why not combine `:hover` and `.focus-visible` classes, keeping either the `opacity` or `background-color`?

Reply:
Oh thanks I didn't think of that !